Detectors/FIT/FV0/workflow/include/FV0Workflow/FV0DataReaderDPLSpec.h

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   FV0DataReaderDPLSpec.h
+
+#ifndef O2_FV0DATAREADERDPLSPEC_H
```

Comment:
To be sure: what is the role of this class? Currently the V0 decoding is done via https://github.com/AliceO2Group/AliceO2/blob/d4991a76d96b5460c02c43561030f133e6ad97f9/Detectors/FIT/FV0/workflow/src/fv0-flp-workflow.cxx#L79-L84, is this class supposed to substitute decoding via FIT-common FITDataReaderDPLSpec ?

Reply:
It seems it's not needed. It's removed now